apps/projects/api.py

```diff
@@ -1,22 +1,27 @@
+from django.db.models import Q
 from rest_framework import permissions
 from rest_framework import viewsets
 
 from adhocracy4.modules.models import Module
+from adhocracy4.projects.enums import Access
 from adhocracy4.projects.models import Project
 
 from .serializers import AppModuleSerializer
 from .serializers import AppProjectSerializer
 from .serializers import ModerationProjectSerializer
 
 
+# FIXME:rename it from AppProjectsViewSet to ProjectViewSet
 class AppProjectsViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = AppProjectSerializer
     permission_classes = (permissions.IsAuthenticated,)
     lookup_field = "slug"
 
     def get_queryset(self):
         return Project.objects.filter(
-            is_draft=False, is_archived=False, is_app_accessible=True
+            Q(access=Access.PUBLIC) | Q(access=Access.SEMIPUBLIC),
```

Comment:
according to @TeaJayyy comment the api should only show current and future projects, this will also include past projects, right?

Reply:
good point, added the extra filter and a relevant test